app/controllers/vm_common.rb

```diff
@@ -1142,6 +1142,9 @@ def replace_right_cell(options = {})
             presenter.update(:form_buttons_div, '')
             presenter.remove_paging.hide(:form_buttons_div)
           end
+        elsif @sb[:action] == 'rename'
+          presenter.update(:form_buttons_div, '')
+          presenter.remove_paging.hide(:form_buttons_div)
```

Comment:
@GilbertCherrie does adding line like `presenter.hide(:form_buttons_div)` doesn't fix it.

Reply:
It seems to be fixing it, however I am just worried this may break other forms

Reply:
If it is fixing that, instead of line no 1146 and 1147, just add that line. As it is in only rename func, it won't break other forms.